### ISSUE TITLE
Auctions for deposits with FRAUD_LIQUIDATION_IN_PROGRESS status not allowed

### ISSUE THREAD

**Issue by tomaszslabon** [keep-network/coverage-pools#99]
Auctions can only be opened for deposits with status `DEPOSIT_LIQUIDATION_IN_PROGRESS_STATE`.
Perhaps auction for deposits with `FRAUD_LIQUIDATION_IN_PROGRESS` status should also be allowed.

Note that in the `purchaseSignerBondsAtAuction` function in tBTC project we have the following check:
```
require(_d.inSignerLiquidation(), "No active auction");```
```
which is:
```
function inSignerLiquidation(DepositUtils.Deposit storage _d) external view returns (bool) {
    return (
        _d.currentState == uint8(States.LIQUIDATION_IN_PROGRESS)
     || _d.currentState == uint8(States.FRAUD_LIQUIDATION_IN_PROGRESS)
    );
}
```

In other words, you can liquidate deposits with `FRAUD_LIQUIDATION_IN_PROGRESS` outside Coverage Pools.
